tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -3967,6 +3999,13 @@ class HLSLExternalSource : public ExternalSemaSource {
     return IsRayQueryBasicKind(GetTypeElementKind(type));
   }
 
+  bool IsHitObjectBasicKind(ArBasicKind kind) {
```

Comment:
This function seems unhelpful, so I don't really think we should have it.

Reply:
Removed in bf3400a99